{{cookiecutter.project_slug}}/package.json

```diff
@@ -35,7 +35,7 @@
     "webpack-merge": "^6.0.1"
   },
   "engines": {
-    "node": "24.10"
+    "node": "25.0"
```

Comment:
I don't think we can upgrade yet: https://devcenter.heroku.com/articles/nodejs-support#supported-node-js-versions

Looks like Heroku only support LTS node version (even numbers)? If that's the case, we should perhaps ignore 25 and skip to 26 directly